### PR TITLE
Reminders Queries

### PR DESCRIPTION
**Pull request by Faultless**
This PR adds a Query and 2 Mutations in order to replace direct API calls with GraphQL calls on the client.

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -301,6 +301,15 @@ export class ZimbraBatchClient {
...
 		});
 
+	public dismissCalendarItem = (appointment: any, task: any) =>
+		this.jsonRequest({
+			name: 'DismissCalendarItemAlarm',
+			body: {
+				appt: appointment,
+				task
+			}
+		}).then(res => !!res);
```
Comment (pl12133) [inline src/batch-client/index.ts]: To cast to a boolean without the extra function declaration you can use the `Boolean` constructor itself: `.then(Boolean)`.

File: src/batch-client/index.ts [hunk 2]
```diff
@@ -635,6 +644,15 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public searchAppointments = (options: SearchOptions) =>
```
Comment (pl12133) [inline src/batch-client/index.ts]: Does the other `search` function not work for appointments? This one looks very similar.

**Comment by Faultless** [comment]
I will need access to merge this, or perhaps someone with write access can do it for me.

**Comment by pl12133** [comment]
@Faultless sure I can merge and publish this.

**Comment by pl12133** [comment]
OK these changes are release as `@zimbra/api-client@7.2.0`.

**Comment by Faultless** [comment]
thank you!

**Pull request merged**